app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb

```diff
@@ -104,6 +111,16 @@ def console_supported?(type)
     return true if type.upcase == 'VNC'
   end
 
+  def console_url
+    crn = ERB::Util.url_encode(ext_management_system.pcloud_crn.values.join(":"))
+    params = URI.encode_www_form(:paneId => "manageiq", :crn => crn)
+    URI::HTTPS.build(:host => "cloud.ibm.com", :path => "/services/power-iaas/#{crn}/server/#{uid_ems}", :query => params)
+  end
+
+  def validate_native_console_support
+    true
+  end
```

Comment:
This is also related to the "remote console" type of support and is not necessary
```suggestion
```

Reply:
I was somehow under the impression that's how it decides to show or route the menu, but you are right. It works without it.